src/_pytest/assertion/util.py

```diff
@@ -342,83 +351,85 @@ def _compare_eq_sequence(
 ) -> List[str]:
     explanation = []
-    diff_left = left - right
-    diff_right = right - left
-    if diff_left:
-        explanation.append("Extra items in the left set:")
-        for item in diff_left:
+    diff_result = result - expected
+    diff_expected = expected - result
+    if diff_result:
+        explanation.append("Extra items in the result set:")
+        for item in diff_result:
             explanation.append(saferepr(item))
-    if diff_right:
-        explanation.append("Extra items in the right set:")
-        for item in diff_right:
+    if diff_expected:
+        explanation.append("Extra items in the expected set:")
+        for item in diff_expected:
             explanation.append(saferepr(item))
     return explanation
 
 
 def _compare_eq_dict(
-    left: Mapping[Any, Any], right: Mapping[Any, Any], verbose: int = 0
+    result: Mapping[Any, Any], expected: Mapping[Any, Any], verbose: int = 0
 ) -> List[str]:
     explanation = []  # type: List[str]
-    set_left = set(left)
-    set_right = set(right)
-    common = set_left.intersection(set_right)
-    same = {k: left[k] for k in common if left[k] == right[k]}
+    set_result = set(result)
+    set_expected = set(expected)
+    common = set_result.intersection(set_expected)
+    same = {k: result[k] for k in common if result[k] == expected[k]}
     if same and verbose < 2:
         explanation += ["Omitting %s identical items, use -vv to show" % len(same)]
     elif same:
         explanation += ["Common items:"]
         explanation += pprint.pformat(same).splitlines()
-    diff = {k for k in common if left[k] != right[k]}
+    diff = {k for k in common if result[k] != expected[k]}
     if diff:
         explanation += ["Differing items:"]
         for k in diff:
-            explanation += [saferepr({k: left[k]}) + " != " + saferepr({k: right[k]})]
-    extra_left = set_left - set_right
-    len_extra_left = len(extra_left)
-    if len_extra_left:
+            explanation += [
+                saferepr({k: result[k]}) + " != " + saferepr({k: expected[k]})
+            ]
+    extra_result = set_result - set_expected
+    len_extra_result = len(extra_result)
+    if len_extra_result:
         explanation.append(
-            "Left contains %d more item%s:"
-            % (len_extra_left, "" if len_extra_left == 1 else "s")
+            "Result contains %d more item%s:"
+            % (len_extra_result, "" if len_extra_result == 1 else "s")
         )
         explanation.extend(
-            pprint.pformat({k: left[k] for k in extra_left}).splitlines()
+            pprint.pformat({k: result[k] for k in extra_result}).splitlines()
         )
-    extra_right = set_right - set_left
-    len_extra_right = len(extra_right)
-    if len_extra_right:
+    extra_expected = set_expected - set_result
+    len_extra_expected = len(extra_expected)
+    if len_extra_expected:
         explanation.append(
-            "Right contains %d more item%s:"
```

Comment:
Same comment as previously here: I think "right" is unambiguously, so if others agree can we revert it?

Reply:
done